src/data/dfinsupp.lean

```diff
@@ -651,6 +651,13 @@ lemma support_subset_iff {s : set ι} {f : Π₀ i, β i} :
 by simp [set.subset_def];
    exact forall_congr (assume i, not_imp_comm)
 
+lemma support_single_eq_zero {i : ι} {b : β i} (hb : b = 0) : (single i b).support = ∅ :=
```

Comment:
This lemma seems a bit silly to me, since the caller can just `rw h`. I think it would be better as:
```suggestion
lemma support_single_eq_zero {i : ι} : (single i (0 : β I)).support = ∅ :=
```
Which we might already have elsewhere

Reply:
Indeed, `by simp [hb]` or `by rw [hb, single_zero, support_zero]` closes the goal, so I don't think this lemma is worth having.